### ISSUE TITLE
Shin Nekketsu Kouha Kunio Tachi no Banka: Slow tempo of the song (SNES, Mint Corp.)

### ISSUE THREAD

**Issue by loveemu** [vgmtrans/vgmtrans#241]
The music in Shin Nekketsu Kouha Kunio Tachi no Banka can be parsed, but when played, it plays at a slower tempo than it actually is.

I haven't looked into the specific cause, but I think it's possible that you're running on a different timer cycle than the driver version used in other Mint games (e.g. Gokinjo Adventure Team).

**Comment by KungFuFurby** [comment]
You're correct. The timer 0 frequency is different. And timer 0 frequencies appear to be set on initialization of the entire sound driver.

Here's my signature set for all of the known games that use the Mint sound driver (there's 12 of them total to look out for):
```
(Program starts at $0334 - Using C4 ?? 3F ?? ??)
(Program starts at $0334 - 5F 0A)
- CB Chara Wars
(Program starts at $0344 - 94 0A)
- Combatribes

(Program starts at $02DC - Using C4 ?? E8 ?? 3F ?? ??)
(Program starts at $02DC - F5 0E)
- Kidou Senshi Gundam - Cross Dimension 0079
- Otoboke Ninja Colosseum
(Program starts at $02DC - EA 0F)
- Lennus II: Fuuin no Shito
- Seifuku Densetsu: Pretty Fighter
- Shounen Ninja Sasuke
(Program starts at $02DC - EE 0F)
- Super Black Bass 2/Bassin's Black Bass
(Program starts at $02DC - F4 0F)
- Bounty Sword
- Gokinjo Boukentai
- Koutetsu no Kishi 3: Gekitotsu Europe Sensen
(Program starts at $02DC - 19 10)
- Gon
- Kidou Senshi Z Gundam: Away to the NewType
(Program starts at $02DC - 34 10)
- Mystic Ark
- Wonder Project J
(Program starts at $02E4 - DF 0F)
- Shin Nekketsu Kouha - Kunio-tachi no Banka
(Program starts at $02EE - 13 0F)
- Shin SD Sengokuden: Taishou Gun Retsuden
(Program starts at $0305 - 96 0F)
- Shien: The Blade Chaser/Shien's Revenge
(Program starts at $037D - DD 13)
- Super Black Bass 3
```
Note that I use a pointer-based approach at the earliest known instance for catching unique builds: under most circumstances this works without much of a problem since the hex comparison then usually tells me that the opcodes are an exact match until the end of the code, but sometimes data in the middle is variable on a per-game basis: those are the ones I have to watch out for and compare by opcode, having to add a note due to the pointers getting thrown off.

Most of these have a timer 0 frequency of $4F set on initialization. The exceptions are...
- CB Chara Wars (Set to $30)
- Combatribes (Set to $28)
- Shien: The Blade Chaser/Shien's Revenge (Set to $3C)
- Shin Nekketsu Kouha - Kunio-tachi no Banka (Set to $3C)
- Super Black Bass 3 (Set to $2D)

All five cases have builds unique to only their games.
